about.html

```diff
@@ -0,0 +1,152 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="links">
+        <ul>
+          <li>
+            <i class="fa-brands fa-facebook-f"></i>
+          </li>
+          <li>
+            <a href="# "> <i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logoout</li>
+        </ul>
+      </div>
+      <nav>
+        <div class="logo">
+          <img src="./images/logo.png" alt="Empowering African women log" />
+        </div>
+        <ul class="menu-links">
+          <i class="close fa fa-times"></i>
+          <li><a class="nav-link" href="./index.html">HOME</a></li>
+          <li><a class="nav-link" href="./about.html">ABOUT</a></li>
+          <li><a class="nav-link" href="">JOIN</a></li>
+          <li><a class="nav-link" href="">SPONSOR</a></li>
+          <li><a class="nav-link" href="">NEWS</a></li>
+          <li><a class="campaign nav-link" href="">CAMPAIGN</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="header">
+      <div class="background">
+        <nav>
+          <i class="fa fa-bars menu"></i>
+        </nav>
+        <div class="hello">
+          <span class="span-hello">"HELLO! AFRICAN WOMEN"</span>
+        </div>
+        <h1 class="about-h1">
+          EMPOWERING AFRICAN WOMEN <br />
+          GLOBAL SUMMIT <br />
+          2023
+        </h1>
+        <p class="description">
+          A joyful celebration believing in the value of empowering women to
+          achieve their destiny by removing barriers that hold them back with
+          people from over 54 countries is taking place in April, Ethiopia.
+        </p>
+
+        <div class="queries">
+          For queries please contact at info@AWEConference.org
+        </div>
+      </div>
+    </section>
+
+    <section class="empowering">
+      <h2>EMPOWERING AFRICAN WOMEN <br />conference discussion issues</h2>
+      <hr class="empowering-hr" />
+
+      <ul class="lists">
+        <li class="list">
+          1. Place women as leaders and give them decision making roles
+        </li>
+        <li class="list">2. More Job Opportunities for Women</li>
+        <li class="list">
+          3. Invest in Women’s Entrepreneurial Ideas, Emotionally and
+          Financially
+        </li>
+        <li class="list">4. Taking Action against Unpaid Labour Work</li>
+        <li class="list">5. Mentoring Women Professionally and Personally</li>
+      </ul>
+    </section>
```

Comment:
- [ ] Kindly edit this section to be more like the original design. As the requirements indicated, you can change the contents of the page but you'll have to stick to the design guidelines (colors, positioning, font style, ...etc) of the original design. Please add a paragraph and then an image below it to match the design.
- [ ] Please change the horizontal line below the title to be shorter and has the same color used in the template.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233102891-d424909f-bc60-406b-ab59-0a1ddc27b992.png) | ![image](https://user-images.githubusercontent.com/62290545/233102772-441a09d6-ea43-419f-a655-38034ecb551b.png) |